sklearn/decomposition/pca.py

```diff
@@ -23,7 +24,7 @@
 
 
 def _assess_dimension_(spectrum, rank, n_samples, n_features):
```

Comment:
why is this function both prefixed *and* suffixed by underscores? Shouldn't it be just the first?